docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+services:
+  go-flare:
+    image: flarefoundation/go-flare
+    container_name: go-flare-node
+    # environment:
```

Comment:
Don't need to give the environment ? is it automatically takes the local config and start running ?

Reply:
Yeah, without env variables, it takes up default values.

Reply:
And does the node automatically cleaned up from the local system when the test process get shut down ?